packages/store/src/actions-stream.ts

```diff
@@ -10,7 +10,8 @@ export const enum ActionStatus {
   Dispatched = 'DISPATCHED',
   Successful = 'SUCCESSFUL',
   Canceled = 'CANCELED',
-  Errored = 'ERRORED'
+  Errored = 'ERRORED',
+  Completed = 'COMPLETED'
```

Comment:
We don't need to define a new completed status, the existing ones infer completion

Reply:
@markwhitfeld I do not agree with you, because COMPLETED status does not promise that he is successful or failed

Reply:
Even tho they infer, I made the COMPLETED status for the readability. It's not crucial, I can remove it. But I my suggestion is to keep it.

Reply:
@splincode These statuses represent the possible lifecycle status values. If we leave this in here then we should have 2 events added to the actions stream when completion happens. Not something that need to be in that stream.

@newbie012 you can leave it in here and I'll follow up your PR with another commit that sorts it the statuses and optimises the performance of the checks in the ofAction pipe. 👍